metadata.c

```diff
@@ -422,6 +422,29 @@ int pv_metadata_rm_usermeta(const char *key)
 	return -1;
 }
 
+static void pv_unescape_data(char *buf, int size)
+{
+	const char chars[] = {'\"', '\\'};
+	int i = -1;
+	int j = 0;
+
+	while (i < size) {
+next:
+		++i;
+		if (buf[i] == '\\') {
+			for (size_t k = 0; k < sizeof(chars); ++k) {
```

Comment:
this is confusing to me, what's the relationship between the chars[] index and sizeof(chars)?

Reply:
As `sizeof` returns the size in bytes so char is a 1-byte data type the `sizeof` is the number of elements.
Now is true, `chars` maybe isn't the best name for that array